userdocs/profiles.dev/docs/intro.md

```diff
@@ -4,8 +4,128 @@ sidebar_position: 1
+  of users who have installed it on their cluster
+- `N` profiles: more than one profile
+- Profiles (capital 'P'): a blanket term for this concept of GitOps native package management
+- Pctl: the CLI tool. Use this to install and manage profiles on your cluster
+:::
+
+---------------------
+
+## Core concepts & Terminology
+
+### GitOps
+
+GitOps is a way of managing your infrastructure and applications so that whole
+system is described declaratively and version controlled (most likely in a Git repository),
+and having an automated process that ensures that the deployed environment matches the state specified in a repository.
+
+For more information, take a look at [“What is GitOps?"](https://www.gitops.tech/#what-is-gitops).
+
+The GitOps tool leveraged by Profiles is Flux. Please refer to [their documentation](https://fluxcd.io/) for more
+information.
+
+### Profile
+
+A Profile is a "package" of Kubernetes deployable objects, known as Artifacts, along with any configurable values.
+An artifact can be one of:
+- Helm Chart;
+- Raw Kubernetes yaml
+- Kustomize patch
+- Another nested Profile
+
+See the [example repo](https://github.com/weaveworks/profiles-examples) for a full collection of working
+profile artifact types.
+
+### pctl
+
+The Profiles CLI.
+
+### Profile Repository
+
+A profile repository holds the definition and other necessary items for an upstream profile
+or profiles. A single repository can hold multiple profiles, separated by directories.
+
+### GitOps Repository
+
+A GitOps repository is one which is synced to your cluster via Flux. The manifests of all
+components applied to your cluster should be managed through this repo.
+
+When installing profiles, the generated manifests will be saved to this repo.
+
+Refer to the [Flux documentation](https://fluxcd.io/) for more detail on how to work with Flux.
+
+### Install
+
+To install a Profile means to generate the required component manifests for a Profile and commit
+them to a GitOps repo. From there, Flux will recognise the changes and apply the manifests.
+
+For more information, see the [Installer documentation](/docs/installer-docs/simple-install).
+
+### Catalog
+
+A Catalog is an in-cluster cache of Profile references. The mechanism allows cluster admins to define
+a list of profiles which may be applied to a cluster, and also provides users with another
+method if installation.
+
+Users can also see when newer versions of profiles are published, as well as query the cache
+for more profiles which may meet their needs.
+
+There is one Catalog per running [Profile Catalog Source Controller](#profile-controller).
+The Catalog is queryable via [pctl](https://github.com/weaveworks/pctl).
+
+Profiles can be added to the Catalog by creating a [`ProfileCatalogSource`](#profile-catalog-source).
+
+### Profile Catalog Source
+
+A `ProfileCatalogSource` is a custom resource through which approved profiles can be managed in the [Catalog](#catalog).
```

Comment:
Should we add an example yaml file here, or link to some kind of description how that looks like? Or will that happen later somewhere? Should we link to that somewhere from here?